### DISCUSSION TITLE
CANT INSTALL v0 ON EXISTING PROJECTS

### DISCUSSION THREAD

**Discussion by vyomakesh09** [vercel/community#7104] [Help] [answered]
### Summary

I'm trying to build on top T3-app which was in next.js but wen we try npx install v0@latest we get an error saying 'npx v0@latest init
Failed to read project configuration. Make sure your project is a Next.js project.' 

my guess is it only works for new projects wen we use create-next-app. If we could scale from here to installing on any project I think it would be a cool.

### Steps to Reproduce

<img width="549" alt="Screenshot 2024-06-06 at 03 47 15" src="https://github.com/vercel/community/assets/54256947/13e45b9a-891f-44e9-b53e-664a55934586">
npx v0@latest init
Failed to read project configuration. Make sure your project is a Next.js project.

**Comment by FightingFalcon** [comment]
also just found this out, any help would be greatly appreciated :)

**Comment by Drakenno** [comment, marked as answer]
you need the components.json file which is created by the shadcn-ui init command:
if youre using npm : npx shadcn-ui@latest init
if any other you might wanna check out : https://ui.shadcn.com/docs/components-json

after this npx v0@latest init will work, atleast it did for me

**Reply by jdhiro** [reply]
Worked for me!

[a comment by an automated account is not shown]
